cmd/tesla-control/commands.go

```diff
@@ -755,35 +762,35 @@ var commands = map[string]*Command{
 			Argument{name: "SEAT", help: "<front|2nd-row|3rd-row>-<left|center|right> (e.g., 2nd-row-left)"},
 			Argument{name: "LEVEL", help: "off, low, medium, or high"},
 		},
-		handler: func(ctx context.Context, acct *account.Account, car *vehicle.Vehicle, args map[string]string) error {
+		handler: func(ctx context.Context, acct *account.Account, vehicle *vehicle.Vehicle, args map[string]string) error {
 			// See SeatPosition definition for controlling backrest heaters (limited models).
-			seats := map[string]vehicle.SeatPosition{
-				"front-left":     vehicle.SeatFrontLeft,
-				"front-right":    vehicle.SeatFrontRight,
-				"2nd-row-left":   vehicle.SeatSecondRowLeft,
-				"2nd-row-center": vehicle.SeatSecondRowCenter,
-				"2nd-row-right":  vehicle.SeatSecondRowRight,
-				"3rd-row-left":   vehicle.SeatThirdRowLeft,
-				"3rd-row-right":  vehicle.SeatThirdRowRight,
+			seats := map[string]action.SeatPosition{
```

Comment:
can we declare the var here instead?
https://github.com/teslamotors/vehicle-command/blob/bce1ebc65c2f8a512f98530aba241692ef6c965c/cmd/tesla-control/commands.go#L28